beamform/deripple.py

```diff
@@ -79,35 +79,47 @@ def deripple(
 
     bw = int(bw) # MAKE SURE IT'S AN INTEGER
     
-    FFFF = FFFF[0, :, 0]
+    FFFF = FFFF[0, :, 0]    
+    
+    # load in coefficients
+    coeffs = np.load(coeffs_fname, mmap_mode="r")
 
-    # ASKAP Parameters
-    N = 1536
-    OS_De = 27.0
-    OS_Nu = 32.0
-    passbandLength = int(((fftLength / 2) * OS_De) / OS_Nu)
+    # calculate pass band Length
```

Comment:
Maybe add an additional explanatory comment here (for what the dimension of FFFF should be)